versioned_docs/version-1.3.0/use-cases/distributed-mongodb-for-multicloud.mdx

```diff
@@ -0,0 +1,1069 @@
+- Install KubeSlice using the `kubeslice-cli` tool. Use the topology file to install the KubeSlice Controller and the 
+  KubeSlice Worker in the k8s-cluster-1 cluster. Install the KubeSlice Worker in the `k8s-cluster-2` and `k8s-cluster-3` clusters 
+  using the same topology file.
+- Create a slice called `demo-slice` across three worker clusters and onboard the `mongodb` namespace.
+- Deploy the MongoDB Kubernetes Operator and Ops manager on the `k8s-cluster-1` cluster,  where the KubeSlice Controller is installed.
+- Deploy the MongoDB across the three clusters using MongoDBMulti.
+- Create a MongoDB replica set using MongoDBMultiCRD.
+- Create the ServiceExport. The service export will be created in all 3 clusters as each cluster will have one replica set.
+
+### Step 1: Clone the Examples Repo
+
+Clone the `examples` repo as it contains all the example YAML files in the 
+`examples/distributed-mongodb` directory. You can use these YAML files to configure MongoDB.
+
+Use the following command to clone the `examples` repo:
+
+```
+git clone https://github.com/kubeslice/examples.git
+```
+
+After cloning the repo, use the files from the `examples/distributed-mongodb` directory.
+
+### Step 2: Merge Kubeconfig Files
+
+1. Replace the `/path/to/kubeconfig/` path to with your local path to access the kubeconfig file.
+
+   ```
+   export KUBECONFIG=/path/to/kubeconfig/cluster-1.config:/path/to/kubeconfig/cluster-2.config:/path/to/kubeconfig/cluster-3.config
+   ```
+   ```
+   kubectl config view --flatten=true > merged.config
+   ```
+
+2. Verify the `merged.config` kubeconfig file using the following command:
+
+   ```
+   export KUBECONFIG=/path/to/kubeconfig/merged.config
+   ```
+
+   ```
+   kubectx
+   ```
+   
+   Expected Output
+
+   ```yaml
+   k8s-cluster-1 # kubeslice controller cluster & kubeslice worker cluster 1 & Mongodb central cluster & mongodb member cluster 1
+   k8s-cluster-2 # kubeslice worker cluster 2 & mongodb member cluster 2
+   k8s-cluster-3 # kubeslice worker cluster 3 & mongodb member cluster 3
+   ```
+
+
+### Step 3: Install KubeSlice
+
+To install KubeSlice using the `kubeslice-cli` tool:
+
+1. Download the latest executable version on your local system depending on your OS from the 
+   [Releases](https://github.com/kubeslice/kubeslice-cli/releases) page. For more information, 
+   see [Install kuebslice-cli](https://kubeslice.io/documentation/open-source/1.3.0/install-kubeslice/kubeslice-cli/install-kubeslice-cli).
+
+   You can use the following command to download the tool:
+   
+   ```yaml
+   sudo curl -fL https://github.com/kubeslice/kubeslice-cli/releases/download/0.4.4/kubeslice-cli-0.4.4-linux-amd64  -o /usr/local/bin/kubeslice-cli 
+   ```
+
+2. Change the file permission using the following command:
+  
+   ```yaml
+   sudo chmod a+x /usr/local/bin/kubeslice-cli
+   ```
+
+3. Prepare the configuration topology file that you will use to install KubeSlice. Use the following template to install the
```

Comment:
Remove the step 3. We don't provide this sample YAML. We just ask them to apply the command. So, refer to ENT use cases.